modules/mssql/mssql_test.go

```diff
@@ -128,25 +128,3 @@ func TestMSSQLServerWithInvalidPassword(t *testing.T) {
 	testcontainers.CleanupContainer(t, ctr)
 	require.NoError(t, err)
 }
-
```

Comment:
I'm removing this test because it's doing exactly the same as another one, using an image that at the moment has no alternative. We can remove, or skip, whatever you prefer.